src/auth/auth.module.ts

```diff
@@ -7,17 +7,18 @@ import { JwtModule } from '@nestjs/jwt';
 import { jwtConstants } from './constants';
 import { JwtStrategy } from './jwt.strategy';
 import { AuthController } from './auth.controller';
+import { JwtRefreshTokenStrategy } from './jwt.refreshtoken.strategy';
 
 @Module({
   imports: [
     UsersModule, 
     PassportModule,
     JwtModule.register({
       secret: jwtConstants.secret,
-      signOptions: { expiresIn: '3600s' },
+      signOptions: { expiresIn: '30s' },  // set to 30s for eaiser demo. Should be 15mins or 1 hour
```

Comment:
I agree with this - I think 15 min is a good time, this should be changed before merging into master